quickstarts/ext-hcp_vault/config.yml

```diff
@@ -0,0 +1,84 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: ext-hcp_vault
+
+# Displayed in the UI (required)
+title: ext-hcp_vault
```

Comment:
This is the display/title that is display within the UI, would you like to update this to a title case display name?

Reply:
@aswanson-nr , I've performed the suggested changes, thanks